.github/workflows/root-ci.yml

```diff
@@ -360,23 +361,21 @@ jobs:
         # Build options: https://root.cern/install/build_from_source/#all-build-options
         include:
           - image: fedora41
-            overrides: ["LLVM_ENABLE_ASSERTIONS=On"]
           - image: fedora42
-            overrides: ["LLVM_ENABLE_ASSERTIONS=On", "CMAKE_CXX_STANDARD=20"]
+            overrides: ["CMAKE_CXX_STANDARD=20"]
           - image: alma8
-            overrides: ["LLVM_ENABLE_ASSERTIONS=On"]
           - image: alma9
-            overrides: ["LLVM_ENABLE_ASSERTIONS=On", "CMAKE_BUILD_TYPE=Debug"]
+            overrides: ["CMAKE_BUILD_TYPE=Debug"]
           - image: ubuntu22
-            overrides: ["imt=Off", "LLVM_ENABLE_ASSERTIONS=On", "CMAKE_BUILD_TYPE=Debug"]
+            overrides: ["imt=Off", "CMAKE_BUILD_TYPE=Debug"]
           - image: ubuntu2404
-            overrides: ["LLVM_ENABLE_ASSERTIONS=On", "CMAKE_BUILD_TYPE=Debug"]
+            overrides: ["CMAKE_BUILD_TYPE=Debug"]
           - image: ubuntu2410
-            overrides: ["LLVM_ENABLE_ASSERTIONS=On", "CMAKE_BUILD_TYPE=Debug"]
+            overrides: ["CMAKE_BUILD_TYPE=Debug"]
           - image: ubuntu2504
-            overrides: ["LLVM_ENABLE_ASSERTIONS=On", "CMAKE_BUILD_TYPE=Debug"]
+            overrides: ["CMAKE_BUILD_TYPE=Debug"]
```

Comment:
This line has a conflict (cxx23 is seeming being now requested).

Reply:
sorry, this is my fault. @hahnjo would you be so kind to solve the conflict? I am approving the PR now.

Reply:
Yes no worries, I knew about the other PR but wanted to make sure that everything (still) passes.